wwe-service-client-api/api-overview/sample.html

```diff
@@ -302,6 +303,22 @@
           command = 'genesys.wwe.service.system.closeToast("wweCustomToast0", succeeded, failed)';
           commandHelp = 'Close a customizable toast.';
           break;
+        case 'system.openDialog':
```

Comment:
Why did you not introduce a custom page for the dialog to demonstrate how to use SCAPI in it?
It could be stored in subfolder named 'assets'.
In 8.5, you built the following code for the dialog page:
`<!DOCTYPE html>
<html>
	<head>
		<title>Custom API Sample: DialogFrame</title>
		<meta http-equiv="X-UA-Compatible" content="IE=edge" />

		<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
		<script src="wwe-service-client-api.js"></script>

		<script>
			// Get the dialog identifier if it exists.
			var dialogId = "",
				matchDialogId = /dialogId=([^&]+)/.exec(window.location.search);
			if(matchDialogId)
				dialogId = matchDialogId[1];


			function closeDialog() {
				genesys.wwe.service.system.closeDialog(dialogId);
			}

		</script>
	</head>
	<body style="text-align: center">
		<img src="https://c.s-microsoft.com/en-us/CMSImages/ImgTwo.jpg?version=2432BB03-C90E-EF03-A2BB-BFA093E1A899" alt="Logo">
		<div>Microsoft Data Privacy Notice</div>
		<div>The message.</div>
		<button class="btn btn-primary" onclick="closeDialog()" style="position: absolute; right: 4px; bottom: 4px;">Close Dialog</button>
	</body>
</html>`

Reply:
Committed.